tests/addTasks.test.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import addItem from '../src/modules/__mock__/addMock.js';
+// import LocalStorage from './mocks/localStorage.js';
+
+describe('addTask', () => {
+  document.body.innerHTML = '<input id="input-task" value="Project completed">';
+  const taskArr = [];
+
+  test('Should return taskArr array with object', () => {
+    expect(addItem(taskArr)).toHaveLength(1);
+  });
+
+  //   test('LocalStorage should be updated', () => {
+  //     expect(localStorage.getItem('data')).toHaveLength(1);
+  //   });
```

Comment:
- [ ] [OPTIONAL] Great job with the tests. Kindly remove the unused pieces of code from the codebase. We are trying to make the codebase as professional as possible. 😎️